sdk/ai/Azure.AI.Inference/src/Telemetry/OpenTelemetryScope.cs

```diff
@@ -0,0 +1,255 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using System.Text.Json;
+using Azure.Core;
+using static Azure.AI.Inference.Telemetry.OpenTelemetryInternalConstants;
+using static Azure.AI.Inference.Telemetry.OpenTelemetryConstants;
+
+namespace Azure.AI.Inference.Telemetry
+{
+    internal class OpenTelemetryScope : IDisposable
+    {
+        /// <summary>
+        /// Activity source is used to save events and log the tags.
+        /// On the ApplicationInsights events are logged to traces table.
+        /// The tags are not logged, but are shown in the console.
+        /// </summary>
+        private static readonly ActivitySource s_chatSource = new(ClientName);
+        /// <summary>
+        /// Add histograms to log metrics.
+        /// On the ApplicationInsights metrics are logged to customMetrics table.
+        /// </summary>
+        private static readonly Meter s_meter = new Meter(ClientName);
+        private static readonly Histogram<double> s_duration = s_meter.CreateHistogram<double>(
+            GenAiClientOperationDurationMetricName, "s", "Measures GenAI operation duration.");
+        private static readonly Histogram<long> s_tokens = s_meter.CreateHistogram<long>(
+            GenAiClientTokenUsageMetricName, "{token}", "Measures the number of input and output token used.");
+
+        private readonly Activity m_activity;
```

Comment:
I wonder what the standard naming is currently for class variables, I see e.g. ChatCompletionsClient.cs uses _endpoint or _pipeline, here m_ or s_

Reply:
Renamed the private class fields from m_variable to _variable. We use s_ prefix for static class fields.